tietokanta/src/main/resources/db/migration/R__Toimenpidekoodit_tehtavaryhmat.sql

```diff
@@ -8,121 +8,59 @@
 
 -- Aja aina R_Toimenpidekoodit ennen näitä inserttejä
 
-INSERT into tehtavaryhma (otsikko, nimi, emo, tyyppi, jarjestys, luotu, luoja, nakyva) VALUES ( '1.0 TALVIHOITO',	'Talvihoito',	NULL,	'ylataso',	1, current_timestamp, (select id from kayttaja where kayttajanimi = 'Integraatio'), TRUE) ON CONFLICT DO NOTHING;
-INSERT into tehtavaryhma (otsikko, nimi, emo, tyyppi, jarjestys, luotu, luoja, nakyva) VALUES ( '1.0 TALVIHOITO',	'Välitaso Talvihoito',	(select id from tehtavaryhma where nimi = 'Talvihoito'),	'valitaso',	1, current_timestamp, (select id from kayttaja where kayttajanimi = 'Integraatio'), FALSE)
-ON CONFLICT (nimi) DO UPDATE set emo = (select id from tehtavaryhma where nimi = 'Talvihoito');
 INSERT into tehtavaryhma (otsikko, nimi, emo, tyyppi, jarjestys, luotu, luoja, nakyva) VALUES ( '1.0 TALVIHOITO',	'Talvihoito (A)',	(select id from tehtavaryhma where nimi = 'Välitaso Talvihoito'),	'alataso',	1, current_timestamp, (select id from kayttaja where kayttajanimi = 'Integraatio'), FALSE)
```

Comment:
Näissä alatason inserteissä on jäljellä näitä viittauksia:
`(select id from tehtavaryhma where nimi = 'Välitaso Talvihoito')`

Eikös esimerkiksi tuon "Välitaso Talvihoito" Insert ole nyt poistettu, joten id:n selectistä tulee silloin toki null?
Sehän ei varmaan tässä haittaa, mutta voisi siistiä noita viittauksia pois taulun riveihin joita ei enää ole.